quattor/client/rpms.pan

```diff
@@ -29,6 +28,11 @@ include { 'rpms/package_default_versions' };
     if(! exists(spma)) {
         SELF[spma] = nlist();
     };
+    ncd = escape('ncm-ncd');
```

Comment:
Hmm, in the docstring of pkg_add I read : If the 'multi' option is not set, the pkg_add function will fail if a previous version is found in the profile. (Multi option is for multiple versions running together, like kernel). so that does not look the same.